.github/workflows/ci.yml

```diff
@@ -46,6 +37,46 @@ jobs:
           - name: CPython 3.9
             tox: py39
             action: 3.9
+          - name: CPython 3.10
+            tox: py310
+            action: '3.10'
+        include:
+          - os:
+              name: Windows
+              runs-on: windows-latest
+            python:
+              name: CPython 3.6
+              tox: py36
+              action: 3.6
+          - os:
+              name: Windows
+              runs-on: windows-latest
+            python:
+              name: CPython 3.7
+              tox: py37
+              action: 3.7
+          - os:
+              name: Windows
+              runs-on: windows-latest
+            python:
+              name: CPytron 3.8
+              tox: py38
+              action: 3.8
+          - os:
+              name: Windows
+              runs-on: windows-latest
+            python:
+              name: CPython 3.9
+              tox: py39
+              action: 3.9
+# Windows 10 not working yet, because there are no wheels yet for it.
+#         - os:
+#               name: Windows
+#               runs-on: windows-latest
+#           python:
+#               name: CPython 3.10
+#               tox: py310
+#               action: '3.10'
```

Comment:
Maybe just add Windows to the matrix (uncomment in this case) like in https://github.com/python-qt-tools/PyQt5-stubs/pull/123/files?  Then add 3.10 separately and you can use an exclude if needed rather than including all the other cases.

(and don't delete the commented out macOS config since it is its own thing)